lib/ClangImporter/ImportType.cpp

```diff
@@ -454,6 +454,9 @@ namespace {
                 ImportHint::OtherPointer};
       }
 
+      if (pointeeQualType->isDependentType())
```

Comment:
The more I think about it, let's see if we can just fix this here and now. Can you add an executable test just to see if the following works? 

Update L473/476:
```
if (!pointeeType || (!pointeeQualType->isDependentType() && Impl.isOverAligned(pointeeQualType))) {
```

...or something similar with a comment about why we don't have to care about the alignment of dependent types.